### ISSUE TITLE
Use pephubclient to upload bed metadata as PEP sample

### ISSUE THREAD

**Issue by khoroshevskyi** [databio/bedboss#31]
Bedboss should utilize PEPHubClient to add bed metadata. For example:

```
phc.sample.add({
    "sample_name": bed_id,
    "genome": "hg38",
    # Add other metadata fields here
})
```
This metadata should be retrieved from the provided PEP to Bedboss. 
Additionally, this line of code should be integrated into bedstat, as all processing occurs there."

**Comment by donaldcampbelljr** [comment]
Made a PEP to capture this information (it is unclear which PEP the pipeline should report his information to):
`databio/allbeds:bedbase`

**Comment by donaldcampbelljr** [comment]
Ok, I've done a first pass on adding this functionality to Bedstat with the caveats:
1. The pephubclient does not exist _yet_
2. Given our test failures, I did not bother writing a test for this either.

**Comment by donaldcampbelljr** [comment]
Ok, I've made the discussed changes. We will still need to validate input as well as remove items from the pep_sample_dict as it will contain attributes that we don't consider metadata. But for now, this would allow for uploading sample to `https://pephub.databio.org/databio/allbeds?tag=bedbase`

**Comment by khoroshevskyi** [comment]
This issue was solved in v0.1.0
